modules/web-mercator/src/web-mercator-utils.ts

```diff
@@ -262,39 +265,69 @@ export function getViewMatrix(options: {
 export function getProjectionParameters(options: {
   width: number;
   height: number;
+  scale: number;
+  center?: number[];
+  offset?: [number, number];
   fovy?: number;
   altitude?: number;
-  pitch?: number;
+  pitch: number;
```

Comment:
Making `pitch` and `scale` non-optional seems like breaking API changes? Was it intentional?